spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe SessionsController do
+  describe "GET new" do
+    it "renders :new template for unauthenticated user" do
+      get :new
+      expect(response).to render_template :new
+    end
+    it "redirects to home page for authenticated user" do
+      session[:user_id] = Fabricate(:user).id
+      get :new
+      expect(response).to redirect_to home_path
+    end
+  end
+
+  describe "POST create" do
+    context "when user is valid" do
+      before do
+        @royal = Fabricate(:user)
+        post :create, email: @royal.email, password: @royal.password
+      end
+      it "sets the session" do
+        expect(session[:user_id]).to eq(@royal.id)
+      end
+      it "redirects to home_path" do
+        expect(response).to redirect_to home_path
+      end
+      it "sets the notice" do
+        expect(flash[:notice]).not_to be_blank
+      end
+    end
+    context "when user is not authenticated" do
+      before do
+        @royal = Fabricate(:user)
```

Comment:
This could be a `let(:royal) { Fabricate(:user) }` right after the context. Then you can do `royal.password + 'kdfsjlkds'`. I think you're doing this in other specs too.